pkg/providers/ovirt/client/ovirt-client.go

```diff
@@ -148,11 +140,15 @@ func (client *richOvirtClient) StopVM(id string) error {
 	select {
 	case success := <-c:
 		if !success {
-			return fmt.Errorf("Failed to stop vm %s", vm.MustName())
+			return fmt.Errorf("Failed to stop vm %s", id)
 		}
 		return nil
 	case <-time.After(vmStopTimeout * time.Minute):
-		return fmt.Errorf("Failed to stop vm %s, current status is %s", vm.MustName(), vm.MustStatus())
+		status := ovirtsdk.VMSTATUS_UNKNOWN
+		if vm != nil {
+			status = vm.MustStatus()
```

Comment:
`MustStatus` could cause panic. Can we use `Status` function?

Reply:
Done.